packages/dx-grid-core/rollup.config.js

```diff
@@ -1,19 +1,29 @@
-import babel from 'rollup-plugin-babel';
```

Comment:
Should we remove this package?

Reply:
TS compiles to ES5, so we don't need polyfills

Reply:
I am sorry, should we remove this package from `devDependencies`?